managing-resources.adoc

```diff
@@ -22,7 +22,7 @@ As an {productname-short} administrator, you can manage the following resources:
 * Custom workbench images
 * Cluster PVC size
 * Cluster storage classes
-* Jupyter notebook servers
+* Notebook servers
```

Comment:
Maybe leave this be. The doc probably has this Admin UI in mind, that is specifically about Jupyter(Lab)

Applications -> Enabled -> Jupyter -> Administration

<img width="1280" alt="image" src="https://github.com/user-attachments/assets/8f008004-7e6a-4387-a061-7c0bc3bcc112">

The bug that's causing the "Error loading component" message is https://issues.redhat.com/browse/RHOAIENG-16434

Reply:
It should be changed to talk about "Workbenches unassociated with Data Science Project" or something like that, but the UI says _Manage notebook servers_ and that's that : shrug :